File: CinemaProject.WebAPI/CinemaProject.BLL/Models/Session.cs
using System;

namespace CinemaProject.BLL.Models
{
    public class Session
    {
        public Guid Id { get; set; }
        public string CinemaName { get; set; }
        public string HallName { get; set; }
        public string FilmName { get; set; }
        public decimal Cost { get; set; }
        public DateTime ShowStart { get; set; }
        public DateTime ShowEnd { get; set; }
        public Guid? HallId { get; set; }
        public Guid? FilmId { get; set; } 

        public Amenity[] AffordableAmenities { get; set; }
    }
}

File: CinemaProject.Angular/src/app/AdminPages/Models/session.ts
import { Amenity, AmenityView } from './amenity';

class Session {
  constructor(
    public id: string,
    public cost: number,
    public showStart: Date,
    public showEnd: Date,
    public hallId: string,
    public cinemaName: string,
    public hallName: string,
    public filmName: string,
    public filmId: string,
    public affortableAmenities: Amenity[]
  ) {}
}

class SessionView {
  constructor(
    public id: string,
    public cost: string,
    public showStart: string,
    public showEnd: string,
    public cinemaName: string,
    public hallName: string,
    public filmName: string,
    public status: string,
    public affortableAmenities: AmenityView[]
  ) {}
}

export { Session, SessionView };
